src/Home/Home.scss

```diff
@@ -45,14 +56,6 @@
       font-family: 'Avenir Heavy';
     }
   }
-
-  .logoutText {

```

Comment:
Why remove this?

Reply:
I moved this to `a.logout` since I changed the `log out` button to be inline with the other links.